analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -145,7 +146,7 @@ open class Npm(
 
     override fun command(workingDir: File?) = if (Os.isWindows) "npm.cmd" else "npm"
 
-    override fun getVersionRequirement(): Requirement = Requirement.buildNPM("6.* - 8.*")
+    override fun getVersionRequirement(): RangesList = RangesListFactory.create("6.* - 8.*")
 
```

Comment:
Note for myself: The NPM ranges are covered by the [XRangeProcessor](https://github.com/semver4j/semver4j/blob/2258ea7b3aa3a43afa8ce41203af9ac844b95d4f/src/main/java/org/semver4j/internal/range/processor/XRangeProcessor.java).